Tasks/FuncToolsInstallerV0/src/utils.ts

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+import tl = require('azure-pipelines-task-lib/task');
```

Comment:
Use a single style here as well. 
`import * as tl from 'azure-pipelines-task-lib'` You can refer to the task functions by referring to the module as well.

Reply:
done